scripts/download_en_asl_gloss_data.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+BASE_DIR=data/raw/lang2gloss/enasl/
+
+wget -P $BASE_DIR https://raw.githubusercontent.com/monkeyhippies/speech2signs-2017-nmt/master/ASLG-PC12/train.tgz
```

Comment:
It's better to wrap variables with double quotes and parenthesis, e.g., "${VAR}". So here $BASE_DIR should be "${BASE_DIR}". Same for other shell scripts.